### ISSUE TITLE
Issues with coverage builds

### ISSUE THREAD

**Issue by ktmf01** [google/oss-fuzz#9984]
It seems flac coverage and introspector builds have failed last 3 times because corpus for a recently added fuzzer cannot be found.

```
Starting Step #5
Step #5: Already have image (with digest): gcr.io/oss-fuzz-base/base-runner
Step #5: [/corpus/fuzzer_tool_metaflac.zip]
Step #5:   End-of-central-directory signature not found.  Either this file is not
Step #5:   a zipfile, or it constitutes one disk of a multi-part archive.  In the
Step #5:   latter case the central directory and zipfile comment will be found on
Step #5:   the last disk(s) of this archive.
Step #5: unzip:  cannot find zipfile directory in one of /corpus/fuzzer_tool_metaflac.zip or
Step #5:         /corpus/fuzzer_tool_metaflac.zip.zip, and cannot find /corpus/fuzzer_tool_metaflac.zip.ZIP, period.
Step #5: Failed to unpack the corpus for fuzzer_tool_metaflac. This usually means that corpus backup for a particular fuzz target does not exist. If a fuzz target was added in the last 24 hours, please wait one more day. Otherwise, something is wrong with the fuzz target or the infrastructure, and corpus pruning task does not finish successfully.
Step #5: ********************************************************************************
Step #5: Code coverage report generation failed.
Step #5: To reproduce, run:
Step #5: python infra/helper.py build_image flac
Step #5: python infra/helper.py build_fuzzers --sanitizer coverage flac
Step #5: python infra/helper.py coverage flac
Step #5: ********************************************************************************
Finished Step #5
ERROR
ERROR: build step 5 "gcr.io/oss-fuzz-base/base-runner" failed: step exited with non-zero status: 1
```

**Another issue** that may be related: When I go to oss-fuzz.com and click "Total coverage" (and in fact go to https://oss-fuzz.com/coverage-report/job/libfuzzer_asan_flac/latest) it redirect me to a 5 weeks old coverage report, from 20230213.

**Comment by oliverchang** [comment]
Sorry for the slow response! This may be related to https://github.com/google/oss-fuzz/issues/9994#issuecomment-1487856069. Will check to see if this one is fixed as well once our quota is approved.

**Comment by oliverchang** [comment]
Ah actually, this may be unrelated, as `flac` is an existing project.

**Comment by oliverchang** [comment]
Looks like we're running into disk space issues for this particular project. I've bumped it a bit -- let's see if that resolves it.

**Comment by ktmf01** [comment]
The issue started when I introduced a new fuzzer.

If FLAC is using a disproportionate amount of resources, please let me know what I can do about it. I am very grateful for the use of oss-fuzz, and I would hate to overburden it.

**Comment by ktmf01** [comment]
Builds no longer failed starting April 1st. It seems this issue is fixed.

Although https://oss-fuzz.com/coverage-report/job/libfuzzer_asan_flac/latest still brings me to the report from 13th of February for some reason.
